packages/flutter/lib/src/material/toggle_buttons.dart

```diff
@@ -790,6 +790,7 @@ class ToggleButtons extends StatelessWidget {
       return MergeSemantics(
         child: Semantics(
           container: true,
+          toggled: isSelected[index],
```

Comment:
Despite the name, this should use the `checked` semantics rather than the `toggled` semantics. `toggled` is read as "Switch", which is incorrect.

Reply:
Thanks for pointing out, fixing shortly. https://github.com/flutter/flutter/pull/113851